app/views/shared/transactions/_table_inside.html.haml

```diff
@@ -58,4 +60,6 @@
           - order_detail.send(:extend, PriceDisplayment)
           = order_detail.wrapped_total
         %td.nowrap= order_detail.order_status
+        - if local_assigns[:show_statements]
+          %td= link_to "Download", account_statement_path(@account, order_detail.statement, format: :pdf) if order_detail.statement
```

Comment:
Can we use just the `order_detail.statement_id` rather than the `statement`? That'll avoid an extra call to the database on each row.